test/Microsoft.Azure.Devices.Edge.Test/PlugAndPlay.cs

```diff
@@ -29,10 +29,10 @@ public class PlugAndPlay : SasManualProvisioningFixture
         const string TestModelId = "dtmi:edgeE2ETest:TestCapabilityModel;1";
         const string LoadGenModuleName = "loadGenModule";
 
-        [TestCase(Protocol.Mqtt, false)]
-        [TestCase(Protocol.Amqp, false)]
-        [TestCase(Protocol.Mqtt, true)]
-        [TestCase(Protocol.Amqp, true)]
+        [TestCase(Protocol.MqttWs, false)]
```

Comment:
Why were these changed?

Reply:
Mentioned this in the description- Fixed some tests to run using Websockets transport (which is generally the case for E2E tests). If this assumption is incorrect, I can revert this. But, this should work for the normal (non-proxy) tests the same.